.gitlab-ci.yml

```diff
@@ -292,10 +295,8 @@ build-malus:
     - echo "polkadot-test-malus = $(cat ./artifacts/VERSION) (EXTRATAG = $(cat ./artifacts/EXTRATAG))"
     - cp -r scripts/* ./artifacts
 
-#### stage:                        build
-
 .check-dependent-project:          &check-dependent-project
-  stage:                           build
+  stage:                           test
```

Comment:
better leave it on the next stage before we're on 14.2, but put in a artificial dependency with `artifacts: false` after something that takes ~ 15 min